monitor/plugins/kubejobs/plugin.py

```diff
@@ -156,3 +159,24 @@ def monitoring_application(self):
 
             self.LOG.log(ex.message)
             raise
+
+    def validate(self, data):
```

Comment:
Shouldn't we add this method in `from monitor.plugins.base` (i.e. should all plugins validate it's info?)
If so, this could be parametrized.

Reply:
No, because the plugin structure isn't consistent, and since we agreed that we aren't going to do structural changes, this was the best solution.

Reply:
@clenimar what do you think?

Reply:
as we discussed before, keeping it consistent through all components would require a significant rewrite of plugin constructors and how they are evoked. this is certainly something we need to tackle in the future.